README.md

```diff
@@ -10,18 +10,17 @@ to choose the right tools as their project evolves. ONNX provides an open source
 It defines an extensible computation graph model, as well as definitions of built-in operators and standard
 data types. Initially we focus on the capabilities needed for inferencing (evaluation).
 
-Caffe2, PyTorch, Microsoft Cognitive Toolkit, Apache MXNet and other tools are developing ONNX support. Enabling interoperability between different
-frameworks and streamlining the path from research to production will increase the speed of innovation in
+Caffe2, PyTorch, Microsoft Cognitive Toolkit, Apache MXNet and other tools are developing ONNX support. Enabling interoperability between different frameworks and streamlining the path from research to production will increase the speed of innovation in
```

Comment:
not easy to catch what is the change here? Could you please tell me what we have changed here?

Reply:
@houseroad I think in the old version, "frameworks" starts a new line which should not.

Reply:
In our brower, I think it will output the same results. @zrphercule Or you mean linting the doc?